### DISCUSSION TITLE
blog/20240129/

### DISCUSSION THREAD

**Discussion by giscus-x** [xaoxuu/blog-comments#43] [Announcements]
# blog/20240129/

其实从年初就知道了这将会是很难熬的一年，事实证明确实如此。虽然 2023 充满了失望和遗憾，但是我对 2024 还是持乐观心态的。

https://xaoxuu.com/blog/20240129/

**Comment by parasolwaddledee** [comment]
2024 加油💪

**Reply by xaoxuu** [reply]
加油加油💪

**Comment by iLincc** [comment]
2024我相信也是开端之年，共勉！

**Reply by xaoxuu** [reply]
共勉！你主页好炫酷啊，好想抄来。

**Comment by Sumorio** [comment]
360环评私以为是资本主义提高拉驴推磨效率的手段罢了。不用太在意360的结果，更在意自我提升和取得的成果就行了。加油

**Reply by xaoxuu** [reply]
应该没那么可怕啦😂，只是工资是我的唯一收入来源，是一切生存活动的基石，所以比较焦虑。

**Comment by easyf12** [comment]
没事，放好心态才能鸽出更多成果！
加油！

**Reply by xaoxuu** [reply]
加油👍

**Comment by kaffa** [comment]
我观察到360环评常会有一种心理补偿效应，就是说被评价人在其它方面获得了认可荣誉和肯定后，评价人会趋向于将相对更高的评价给到360评价之外未获荣誉和肯定的人。这就是一种诡计，这些绩效工具被设计出来，很多时候是被草草启用缺少全面的认识而背离初衷的。在HRM领域很少能看到技术动作很成效都很好的方案。

**Reply by xaoxuu** [reply]
我就是一开始得到了很高的评价，后来我给出的评分也都很高，再然后就出现了我得到的平均分比我给出的最低分还要明显低约10%～20%，这个给我的感觉就是挺随机的，每个人有每个人的评分标准，参与量少的时候就是个笑话：对参与者而言，根本不知道评价人的评分标准，对后台而言，可以分析往年数据，得出每个人的评分喜好，从而对结果进行较为精准的控制，很有操作空间，是否进行操作全看自觉了。

**Comment by W4J1e** [comment]
360环评这些词汇我都没接触过，可能只是表达方式不一样，我感觉职场的各种评价，绩效啥的都弄得我云里雾里。
顺便，催一下 volantis 6.x 的开发，我好喜欢渲染图那个效果！

**Reply by xaoxuu** [reply]
volantis 6 是今年的重点工作
